libs/platform/experience/src/mixins/layout.mixin.ts

```diff
@@ -47,6 +50,8 @@ export const LayoutMixin = <T extends Type<LitElement & LayoutAttributes>>(
     @signalProperty({ type: Boolean }) bleed?: boolean;
     @signalProperty({ type: Boolean }) sticky?: boolean;
     @signalProperty({ type: Boolean }) overlap?: boolean;
+    @signalProperty({ type: Boolean }) divider?: boolean;
+    @property({ type: Boolean, reflect: true }) vertical?: boolean;
```

Comment:
No need to afaik, as the vertical is only used as a reflected property in various layout rules rather than being observed to load style rules. I did however improved a few small things